packages/spinners/lib/spinner.ts

```diff
@@ -45,55 +49,54 @@ export class Spinner {
       return;
     }
     spinnies.remove(this.name);
-    spinnies.checkIfActiveSpinners();
```

Comment:
Sorry, as someone not familiar with all the spinner changes, what's the significance of this change?

Reply:
The line that was removed was a workaround for a known issue in the official `spinnies` module. I fixed that issue in our fork, and removed the workaround.

Reply:
Bug in question was: jcarpanelli/spinnies#34

Reply:
OK, thanks!